pkg/controller/restore_finalizer_controller.go

```diff
@@ -392,3 +401,48 @@ func needPatch(newPV *v1.PersistentVolume, pvInfo *volume.PVInfo) bool {
 
 	return false
 }
+
+// WaitRestoreExecHook waits for restore exec hooks to finish then update the hook execution results
+func (ctx *finalizerContext) WaitRestoreExecHook() (errs results.Result) {
+	log := ctx.logger.WithField("restore", ctx.restore.Name)
+	log.Info("Waiting for restore exec hooks starts")
+
+	// wait for restore exec hooks to finish
```

Comment:
Is it possible to use `wait.PollInfinite`?

Reply:
Sure.